src/stactools/sentinel1/grd/stac.py

```diff
@@ -76,9 +75,14 @@ def create_item(
     item.common_metadata.constellation = SENTINEL_CONSTELLATION
 
     # s1 properties
-    shape = get_shape(metalinks, read_href_modifier)
+    extra_properties = {}
+
+    if not os.getenv("SKIP_SHAPE"):
+        shape = get_shape(metalinks, read_href_modifier)
+        extra_properties["s1:shape"] = shape
```

Comment:
The use and purpose of this environment variable should be documented, or (preferably) moved to a argument to the `create_item` function. Relying on environment variables can be fragile.